
#*************** ensenso_nx node and capture parameters ************************

#serial number (i.e. 171416 or 171417)
serial_number: "171416"

#run mode: run mode: 0->SERVER, 1->PUBLISHER
run_mode: 0

#loop rate in hz (good values: 10 when SERVER, below 1 when PUBLISHER)
rate: 20

#frame name to which point cloud is referenced (relative, without namespace)
frame_name: "camara_lens_left"

#capture autoexposure [t/f]
auto_exposure: True

#capture exposure time [ms]
exposure_time: 0

#Dense cloud: 1->Node returns a dense cloud (ordered). 0->Node returns a non-dense cloud (unordered)
dense_cloud: 1
